docs/settings/alert-action-settings.asciidoc

```diff
@@ -130,6 +130,9 @@ A list of allowed email domains which can be used with the email connector. When
 
 WARNING: This feature is available in {kib} 7.17.4 and 8.3.0 onwards but is not supported in {kib} 8.0, 8.1 or 8.2. As such, this setting should be removed before upgrading from 7.17 to 8.0, 8.1 or 8.2. It is possible to configure the settings in 7.17.4 and then upgrade to 8.3.0 directly.
 
+`xpack.actions.enableFooterInEmail` {ess-icon}::
```

Comment:
Should we also add this to the docker allowlist?

Reply:
yup - opened https://github.com/elastic/cloud/issues/113918 to track

Reply:
The docker allowlist is here `src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker` :)

Reply:
merged that fix here: https://github.com/elastic/kibana/pull/155076